design/one-pager-definition-of-done.md

```diff
@@ -0,0 +1,46 @@
+# Definition of done
+
+* Owner: Daniel Suskin (@suskin)
+* Reviewers: Crossplane Maintainers
+* Status: Draft
+
+## Introduction
+
+*Why does this document exist?*
+
+The goal of this document is to answer the questions:
+
+* As a developer, when should I consider a chunk of work to be complete?
+* As a reviewer, what should I be looking for to verify that a chunk of work is
+  complete?
+
+## Definition of work
+
+Work means a contribution to the project, typically code.
+
+## Definition of done
+
+The following criteria **MUST** be met for work to be considered done:
+
+* Code is reviewed and approved
+* Code passes linting (use `make reviewable` to lint)
+* All error conditions are reported into the log or as an event, as
+  appropriate
+* Code is tested
+
+Additionally, the following criteria **SHOULD** be met for the work to
+be considered done, as they are best practices:
+
+* Code is tested with an integration test
+* New behavior is documented in the documentation that will go onto the
+  crossplane website
+* Examples have been updated to reflect any changes
+
+## Applying definition of done
+
+The definition of done can be applied in the following ways:
```

Comment:
> As a reviewer, what should I be looking for to verify that a chunk of work is complete?

Given the above, do we prefer all work to be tracked in a PR? Put otherwise, is it acceptable to add code and documentation in separate PRs? Is it acceptable to add code and tests in separate PRs? My preference would be to encourage that all work - code, tests, docs, etc - be grouped (potentially as separate commits) in one PR.

Reply:
https://github.com/crossplaneio/crossplane/pull/931/files#diff-f31a09f3f47ef35a6f5d5b83aa497071R83-R90 specifies that all work (docs, examples, testing, code) related to a change or set of changes in Crossplane must be included in a single PR.